elasticsearch_metrics/management/commands/sync_metrics.py

```diff
@@ -1,12 +1,37 @@
-from django.core.management.base import BaseCommand
+from django.core.management.base import BaseCommand, CommandError
+
+from elasticsearch_metrics.registry import registry
 
 
 class Command(BaseCommand):
```

Comment:
When would we run this management command? When the app is instantiated?

Reply:
Yeah, we'd probably do it the same as `manage.py migrate`: automatically run it on app startup

We technically only run `migrate` automatically for staging servers, but this command is much less likely to be destructive, so I think we can just run this command on startup for all deployments.